cumulus_library/cli.py

```diff
@@ -113,73 +113,126 @@ def export_all(self, study_dict: Dict):
             self.export_study(study_dict[key])
 
 
-def get_study_dict() -> Dict[str, PosixPath]:
+def get_abs_posix_path(path: str) -> PosixPath:
+    """Conveince method for hanlding abs vs rel paths"""
+    if path[0] == "/":
+        return Path(path)
+    else:
+        return Path(Path.cwd(), path)
+
+
+def create_template(path: str) -> None:
+    """Creates a manifest in target dir if one doesn't exist"""
+    abs_path = get_abs_posix_path(path)
+    manifest_path = Path(abs_path, "manifest.toml")
+    if manifest_path.exists():
+        sys.exit(f"A manifest.toml already exists at {abs_path}, skipping creation")
+    template_path = Path(
+        Path(__file__).resolve().parents[0], "studies/template/manifest.toml"
+    )
+    abs_path.mkdir(parents=True, exist_ok=True)
+    manifest_path.write_bytes(template_path.read_bytes())
+
+
+def get_study_dict(alt_paths: List) -> Optional[Dict[str, PosixPath]]:
     """Convenience function for getting directories in ./studies/
 
     :returns: A list of pathlib.PosixPath objects
     """
     manifest_studies = {}
-    library_path = Path(__file__).resolve().parents[0]
-    for path in Path(f"{library_path}/studies").iterdir():
-        if path.is_dir():
-            manifest_studies[path.name] = path
+    paths = [Path(Path(__file__).resolve().parents[0], "studies")]
+    if alt_paths is not None:
+        paths = paths + alt_paths
+    for parent_path in paths:
+        for child_path in parent_path.iterdir():
+            if child_path.is_dir():
+                manifest_studies[child_path.name] = child_path
+            elif child_path.name == "manifest.toml":
+                manifest_studies[parent_path.name] = parent_path
     return manifest_studies
 
 
 def run_cli(args: Dict):  # pylint: disable=too-many-branches
```

Comment:
lol I didn't see this pylint disable before -- it may honestly be trying to help here

Reply:
oh that was from when this was :much: grosser, and i meant to trim it - i removed it and it is now not complaining.

Reply:
Hmm, does our linting config turn off warning about unnecessary lint disables? That's a nice one.

Reply:
`useless-suppression` is in the disabled list.

Reply:
Why, Google. Such a helpful mistake-catcher, and keeps you from backsliding. I don't feel strongly enough to advocate for a delta, but I wouldn't hate it.